lemur/plugins/lemur_linuxdst/remote_host.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+from lemur.certificates import service
+import os
+
+
+def createCert(name, tempFolder, exportType):
+
+    lemCert = service.get_by_name(name)
+    if not os.path.exists(tempFolder):
+        os.mkdir(tempFolder)
+    if not os.path.exists('{0}/{1}'.format(tempFolder, lemCert.cn)):
+        os.mkdir('{0}/{1}'.format(tempFolder, lemCert.cn))
+    certFile = '{0}/{1}/cert.pem'.format(tempFolder, lemCert.cn)
+    keyFile = '{0}/{1}/priv.key'.format(tempFolder, lemCert.cn)
```

Comment:
I was able to resolve it with : 
![image](https://cloud.githubusercontent.com/assets/704271/24318979/3c2fb446-10e6-11e7-927d-cbc074baf261.png)
my concern is that it's automatically accepting the host key :-(